### ISSUE TITLE
Suppressing console.log

### ISSUE THREAD

**Issue by perry-garvin** [f-elix/vue-xstate-plugin#4]
I'd also be keen to adjust this so that users could optionally suppress the console log about the machine starting. Maybe a "inDevelopment" flag?

**Comment by f-elix** [comment]
Great idea. Would you mind submitting a pull request for this as well? I'm kind of short for time these days, so I would really appreciate it.

**Comment by perry-garvin** [comment]
Yes, I'll work on this and a few other inoffensive tweaks. If this feels like too much of a burden for you I can help you maintain. More soon!
